src/fetchers/Graph/GraphFetcher.ts

```diff
@@ -29,17 +29,20 @@ export abstract class GraphFetcher extends Fetcher {
       Date.now() - this._lastUpdateTimestamp > this.CACHE_DURATION
     ) {
       this._lastUpdateTimestamp = Date.now();
-      this._lastIndexedBlock = fetchSubgraph<{ _meta: { block: { number: number } } }>(
-        GRAPH_URL,
-        BLOCK_QUERY
-      ).then((res: GraphResult<{ _meta: { block: { number: number } } }>) => {
-        if (!res.data) {
-          // eslint-disable-next-line no-console
-          console.error(`Error while fetching graph: ${JSON.stringify(res.errors)}`); //Silently fail if graph error
-          return;
+      this._lastIndexedBlock = fetchSubgraph<{
+        _meta: { block: { number: number } };
+      }>(GRAPH_URL, BLOCK_QUERY).then(
+        (res: GraphResult<{ _meta: { block: { number: number } } }>) => {
```

Comment:
Why do we need to repeat the type?

Reply:
idk, not related to this PR, this is just a lint issue